modules/ossm-access-grafana.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-observability.adoc
+
+[id="ossm-access-grafana_{context}"]
+= Accessing Grafana
+
+Grafana is an analytics tool that allows you to view, query, and analyze your service mesh metrics. To access Grafana, do the following:
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Click the *Project* menu and choose the `istio-system` project from the list.
+
+. Click *Routes*.
+
+. Click the location link for the Grafana row.
```

Comment:
It seems like there is no UI label, such as *Location*, that you could mention here. Right?

Reply:
Right, it's a column in a table. I've made a small change.